### DISCUSSION TITLE
Implement "Sound Balance" /  "More Engine sound" from Arcade UI for Coast 2 Coast modes.

### DISCUSSION THREAD

**Discussion by thegreatprophetzarquon** [emoose/OutRun2006Tweaks#230] [Suggestions]
### Describe the feature you'd like to see added

@ksdomino's [OR2LouderEngineSound](https://github.com/ksdomino/OR2LouderEngineSound) mod manually edits some of the game's audio files to make the engine sound louder in comparison. It describes the quiet engine sound as an oversight or bug, but I think it's a deliberate change made for Outrun 2 SP to keep it from getting too monotonous[^1].

![Screenshot 2024-10-13 002039](https://github.com/user-attachments/assets/28e80408-df96-4589-9aeb-5ab7f267b31b)
Outrun 2 SP and the SP arcade mode in Coast 2 Coast provide an option to have a louder engine in the selection UI. You hold down "X" and can select between "More BGM," "Normal balance" and "More Engine sound" (which seems to mimic what the original OR2 and the 1st Xbox port sound like). This is not an option when selecting any Coast 2 Coast mission or mode.

I'm fine with the default sound mix, but would it be possible to hook into whatever the arcade UI enables when selecting "more engine sound" and have it as an ini setting for Coast 2 Coast? Then there would be no need to manually mod the audio and play with the balance for those who prefer it.

[^1]: I could be wrong: I don't have the most sensitive ears, but the audio levels [here](https://www.youtube.com/watch?v=UDLzVRNK35o) from an Outrun 2 SP board seem to line up with base Coast 2 Coast, while the original Xbox port (largely based on the first arcade version) accordingly has a louder engine noise.

**Comment by Tenome** [comment]
This would be a nice feature.

**Comment by Zoukambada** [comment]
The engine sound mod is great, so... +1 to this feature.
